tests/test_icml_conference.py

```diff
@@ -1309,3 +1328,114 @@ def test_decision_stage(self, openreview_client, helpers):
         assert openreview_client.get_invitation('ICML.cc/2023/Conference/Submission3/-/Decision')
         assert openreview_client.get_invitation('ICML.cc/2023/Conference/Submission4/-/Decision')
         assert openreview_client.get_invitation('ICML.cc/2023/Conference/Submission5/-/Decision')
+
+    def test_forum_chat(self, openreview_client, helpers):
+
+        openreview_client.post_invitation_edit(
+            invitations='ICML.cc/2023/Conference/-/Edit',
+            readers = ['ICML.cc/2023/Conference'],
+            writers = ['ICML.cc/2023/Conference'],
+            signatures = ['ICML.cc/2023/Conference'],
+            invitation = openreview.api.Invitation(
+                id = 'ICML.cc/2023/Conference/-/Submission',
+                reply_forum_views = [
+                    {
+                        "id": "all",
+                        "label": "All"
+                    },
+                    {
+                        "id": "reviewers-chat",
+                        "label": "Reviewers Chat",
+                        "filter": "invitation:ICML.cc/2023/Conference/Submission${note.number}/-/Chat,ICML.cc/2023/Conference/Submission${note.number}/-/Official_Review",
+                        "nesting": 2,
+                        "sort": "date-asc",
+                        "layout": "chat",
+                        "live": True,
+                        "expandedInvitations": ["invitation:ICML.cc/2023/Conference/Submission${note.number}/-/Chat"]
+                    }
+                ]
+            )
+        )
+
+        submission_invitation = openreview_client.get_invitation('ICML.cc/2023/Conference/-/Submission')
+        assert len(submission_invitation.reply_forum_views)
+
+        submission = openreview_client.get_notes(invitation='ICML.cc/2023/Conference/-/Submission', number=1)[0]
+
+        openreview_client.post_invitation_edit(
+            invitations='ICML.cc/2023/Conference/-/Edit',
+            readers = ['ICML.cc/2023/Conference'],
+            writers = ['ICML.cc/2023/Conference'],
+            signatures = ['ICML.cc/2023/Conference'],
+            invitation = openreview.api.Invitation(
+                id = 'ICML.cc/2023/Conference/Submission1/-/Chat',
+                readers = ['everyone'],
+                writers = ['ICML.cc/2023/Conference'],
+                signatures = ['ICML.cc/2023/Conference'],
+                invitees = ['ICML.cc/2023/Conference/Program_Chairs', 'ICML.cc/2023/Conference/Submission1/Reviewers'],
```

Comment:
Can we add Area Chairs to invitees as well?

Reply:
done.